src/services/firebase.js

```diff
@@ -66,4 +65,9 @@ export const startFirebaseUI = function (elementId) {
         ui.start(elementId, uiConfig)
     })
 };
-export const db = firebase.database();
\ No newline at end of file
+
+export function getCurrentUser() {

```

Comment:
Similarly, we will not need this once authentication is finished and we keep track of user through React Context. Good to keep for now.